packages/mgt/src/components/mgt-login/mgt-login.ts

```diff
@@ -97,6 +99,49 @@ export class MgtLogin extends MgtTemplatedComponent {
   public connectedCallback() {
     super.connectedCallback();
     this.addEventListener('click', e => e.stopPropagation());
+    LocalizationHelper.onUpdated(this.handleLocalizationChanged);
```

Comment:
The registering/unregistering of localization events would make sense to go in the BaseComponent too.  You can add `updateDirection` as a virtual method in the base and then override as needed in each component class.

Reply:
Thanks I think I figured it out! Now all the methods important to localization are handled there. The component is only reading the strings.